nixos/modules/services/networking/stubby.nix

```diff
@@ -30,7 +30,7 @@ in {
       settings = mkOption {
         type = types.attrsOf settingsFormat.type;
         example = lib.literalExpression ''
-          pkgs.stubby.passthru.settingsExample // {
+          (builtins.fromJSON (builtins.readFile pkgs.stubby.stubbyExampleJson)) // {
```

Comment:
Isn't this an IFD?

Reply:
Yes it is, good that you noticed. I had forgotten that and thought I was optimising something. I will push a fix when I get a chance.